pkg/apis/org/v1/che_types.go

```diff
@@ -638,25 +569,25 @@ type CheClusterStatus struct {
 	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors.displayName="Keycloak Admin Console URL"
 	// +operator-sdk:gen-csv:customresourcedefinitions.statusDescriptors.x-descriptors="urn:alm:descriptor:org.w3:link"
 	KeycloakURL string `json:"keycloakURL"`
-	// Public URL to the Devfile registry
+	// Public URL to the devfile registry.
 	// +optional
 	DevfileRegistryURL string `json:"devfileRegistryURL"`
-	// Public URL to the Plugin registry
+	// Public URL to the Plugin registry.
```

Comment:
Maybe "Plugin" should start with lowercase too?

Reply:
right